### DISCUSSION TITLE
关于损坏文件的补档

### DISCUSSION THREAD

**Discussion by ProletRevDicta** [ProletRevDicta/Prolet#2335] [Announcements]
@hubred-rev 同志制作了pdf后删去了部分原压缩包，但是存在pdf损坏的情况(如/毛远新著述摘编（“‘四人帮’‘死党’毛远新的‘反动’言论摘编”，“中共辽宁省委清查‘四人帮’罪行小组” 1978年6月）.pdf)，请帮助给予核查与补档，否则反而造成损失

**Comment by ProletRevDicta** [comment]
另，部分揭批查材料拟先作目录，再逐渐上录

**Comment by moe-rika** [comment]
其实文件是删不掉的，找到原来的commit就行了

**Comment by moe-rika** [comment]
比如说合并之前的commit
https://github.com/ProletRevDicta/Prolet/tree/07cdeb9a3a8c3f9628248bdebecd94f575550edd

**Comment by moe-rika** [comment]
整理主要是@hubred-rev 同志在做，我只是提议。

**Comment by moe-rika** [comment]
我担心这个同志会被GitHub判定为spamming然后被删号，不过commit仍然会保留。

**Reply by ProletRevDicta** [reply]
感谢告知！

**Comment by moe-rika** [comment]
[毛远新著述摘编（“‘四人帮’‘死党’毛远新的‘反动’言论摘编”，“中共辽宁省委清查‘四人帮’罪行小组” 1978年6月）.7z](https://github.com/ProletRevDicta/Prolet/blob/07cdeb9a3a8c3f9628248bdebecd94f575550edd/%E6%AF%9B%E8%BF%9C%E6%96%B0%E8%91%97%E8%BF%B0%E6%91%98%E7%BC%96%EF%BC%88%E2%80%9C%E2%80%98%E5%9B%9B%E4%BA%BA%E5%B8%AE%E2%80%99%E2%80%98%E6%AD%BB%E5%85%9A%E2%80%99%E6%AF%9B%E8%BF%9C%E6%96%B0%E7%9A%84%E2%80%98%E5%8F%8D%E5%8A%A8%E2%80%99%E8%A8%80%E8%AE%BA%E6%91%98%E7%BC%96%E2%80%9D%EF%BC%8C%E2%80%9C%E4%B8%AD%E5%85%B1%E8%BE%BD%E5%AE%81%E7%9C%81%E5%A7%94%E6%B8%85%E6%9F%A5%E2%80%98%E5%9B%9B%E4%BA%BA%E5%B8%AE%E2%80%99%E7%BD%AA%E8%A1%8C%E5%B0%8F%E7%BB%84%E2%80%9D%201978%E5%B9%B46%E6%9C%88%EF%BC%89.7z)

毛远新关于教育革命的部分讲话（“辽宁省教育局大批判组”1976年12月整理的黑材料“‘四人帮’死党毛远新‘破坏’教育革命言论摘编”）.7z.

https://github.com/ProletRevDicta/Prolet/blob/07cdeb9a3a8c3f9628248bdebecd94f575550edd/%E6%AF%9B%E8%BF%9C%E6%96%B0%E5%85%B3%E4%BA%8E%E6%95%99%E8%82%B2%E9%9D%A9%E5%91%BD%E7%9A%84%E9%83%A8%E5%88%86%E8%AE%B2%E8%AF%9D%EF%BC%88%E2%80%9C%E8%BE%BD%E5%AE%81%E7%9C%81%E6%95%99%E8%82%B2%E5%B1%80%E5%A4%A7%E6%89%B9%E5%88%A4%E7%BB%84%E2%80%9D1976%E5%B9%B412%E6%9C%88%E6%95%B4%E7%90%86%E7%9A%84%E9%BB%91%E6%9D%90%E6%96%99%E2%80%9C%E2%80%98%E5%9B%9B%E4%BA%BA%E5%B8%AE%E2%80%99%E6%AD%BB%E5%85%9A%E6%AF%9B%E8%BF%9C%E6%96%B0%E2%80%98%E7%A0%B4%E5%9D%8F%E2%80%99%E6%95%99%E8%82%B2%E9%9D%A9%E5%91%BD%E8%A8%80%E8%AE%BA%E6%91%98%E7%BC%96%E2%80%9D%EF%BC%89.7z.001



https://github.com/ProletRevDicta/Prolet/blob/07cdeb9a3a8c3f9628248bdebecd94f575550edd/%E6%AF%9B%E8%BF%9C%E6%96%B0%E5%85%B3%E4%BA%8E%E6%95%99%E8%82%B2%E9%9D%A9%E5%91%BD%E7%9A%84%E9%83%A8%E5%88%86%E8%AE%B2%E8%AF%9D%EF%BC%88%E2%80%9C%E8%BE%BD%E5%AE%81%E7%9C%81%E6%95%99%E8%82%B2%E5%B1%80%E5%A4%A7%E6%89%B9%E5%88%A4%E7%BB%84%E2%80%9D1976%E5%B9%B412%E6%9C%88%E6%95%B4%E7%90%86%E7%9A%84%E9%BB%91%E6%9D%90%E6%96%99%E2%80%9C%E2%80%98%E5%9B%9B%E4%BA%BA%E5%B8%AE%E2%80%99%E6%AD%BB%E5%85%9A%E6%AF%9B%E8%BF%9C%E6%96%B0%E2%80%98%E7%A0%B4%E5%9D%8F%E2%80%99%E6%95%99%E8%82%B2%E9%9D%A9%E5%91%BD%E8%A8%80%E8%AE%BA%E6%91%98%E7%BC%96%E2%80%9D%EF%BC%89.7z.002

毛远新1966-1974年的部分讲话文章.7z

https://github.com/ProletRevDicta/Prolet/blob/07cdeb9a3a8c3f9628248bdebecd94f575550edd/%E6%AF%9B%E8%BF%9C%E6%96%B01966-1974%E5%B9%B4%E7%9A%84%E9%83%A8%E5%88%86%E8%AE%B2%E8%AF%9D%E6%96%87%E7%AB%A0.7z

**Reply by ProletRevDicta** [reply]
感谢！

**Comment by moe-rika** [comment]
补档https://github.com/ProletRevDicta/Prolet/issues/2704
